src/services/observables/pools.ts

```diff
@@ -201,15 +200,15 @@ const buildPoolV3Object = (
     programs
   );
 
-  const tradingFeesApr24h = calcApr(
-    new BigNumber(apiPool.fees24h.usd).minus(apiPool.networkFees24h.usd),
-    stakedBalance.usd
-  );
-  const tradingFeesApr7d = calcApr(
-    new BigNumber(apiPool.fees7d.usd).minus(apiPool.networkFees7d.usd),
-    stakedBalance.usd,
-    true
-  );
+  const tradingFeesApr24h = 0; // calcApr(
```

Comment:
Should the comments stay ? Maybe create an issue if it's meant to be put back later

Reply:
I wanted to leave it there so that anyone coming back and wondering why the text doesn't match the number will have a clue